### PR TITLE
net volume

### LINKED ISSUE #905: show net volume not just bidirection volume
currently we show only bidirectional volume, which is `in + out` which is useless for most people most of the time

we should show net volume (a.k.a. "profit") which is usually what people care about, as `in - out` on the volume tab

### PR DESCRIPTION
**Pull request by rouzwelt**
## Motivation
resolves #905 

## Solution
Screenshot 2024-10-05 at 10 47 49 PM

## Checks

By submitting this for review, I'm confirming I've done the following:
- [x] made this PR as small as possible
- [x] unit-tested any new functionality
- [x] linked any relevant issues or PRs

### CODE REVIEW

File: tauri-app/src/lib/components/tables/OrderVaultsVolTable.svelte [hunk 1]
```diff
@@ -54,8 +54,8 @@
     <TableBodyCell tdClass="break-all py-2" data-testid="total-out">
       {formatUnits(BigInt(item.totalOut), Number(item.token.decimals ?? 0))}
     </TableBodyCell>
-    <TableBodyCell tdClass="break-all py-2" data-testid="total-vol">
-      {formatUnits(BigInt(item.totalVol), Number(item.token.decimals ?? 0))}
```
Comment (thedavidmeister) [inline tauri-app/src/lib/components/tables/OrderVaultsVolTable.svelte:58]: i don't understand why we removed total volume? i was expecting to see both total and net

**Review by thedavidmeister — CHANGES_REQUESTED** [review]

File: tauri-app/src/lib/components/tables/OrderVaultsVolTable.test.ts [hunk 2]
```diff
@@ -105,12 +107,12 @@ test('renders table with correct data', async () => {
 
   await waitFor(async () => {
     // get total vols
-    const rows = screen.getAllByTestId('total-vol');
+    const rows = screen.getAllByTestId('net-vol');
 
     // checking the total vols
     for (let i = 0; i < mockVaultsVol.length; i++) {
       const display = formatUnits(
-        BigInt(mockVaultsVol[i].totalVol),
```
Comment (thedavidmeister) [inline tauri-app/src/lib/components/tables/OrderVaultsVolTable.test.ts:113]: we lost the test for total vol?

**Review by thedavidmeister — CHANGES_REQUESTED** [review]

File: tauri-app/src/lib/components/tables/OrderVaultsVolTable.test.ts (hunk 2 above)
Comment (rouzwelt) [reply tauri-app/src/lib/components/tables/OrderVaultsVolTable.test.ts]: fixed

Comment (rouzwelt) [reply tauri-app/src/lib/components/tables/OrderVaultsVolTable.test.ts]: @thedavidmeister added

**Review by thedavidmeister — APPROVED** [review]

**Pull request merged**